dm/templates/folder/folder.py

```diff
@@ -22,15 +22,20 @@ def generate_config(context):
 
     resources = []
     out = {}
-    for folder in context.properties.get('folders', []):
+    for i, folder in enumerate(

```

Comment:
The variable `i` is not used in the section below, please revert lines 27-29.
Adam and I had a chat about the use of the 10 character truncated SHA-1 digest used below and agree that this is sufficient, since the chance of a collision occurring within a single resource is extremely low (Git uses 7 characters for the truncated notation by default).